cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
In the `validate` method of `TicketSerializer`, it's not clear where the `validate_ticket` method comes from. If it's a class method of `Ticket`, it's better to call it using `self.instance.__class__.validate_ticket` to make it clear.